src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -454,11 +454,16 @@ private function buildTableRows($rows)
      * @param int   $line
      *
      * @return array
+     *
+     * @throws InvalidArgumentException
      */
     private function fillNextRows(array $rows, $line)
     {
         $unmergedRows = array();
         foreach ($rows[$line] as $column => $cell) {
+            if (null !== $cell && !is_string($cell) && !$cell instanceof TableCell) {
```

Comment:
I would keep allowing scalars. Passing integers or floats works well today, and there is no reason to break it (and even less reason to break it in a patch release)